src/vxworks/mod.rs

```diff
@@ -2167,6 +2016,9 @@ extern {
         iov: *const ::iovec,
         iovcnt: ::c_int,
     ) -> ::ssize_t;
+
+    // randomNumGen.h
+    pub fn randBytes(buf: *mut u8, length: i32) -> i32;
```

Comment:
For consistency you should use `pub fn randBytes(buf: *mut ::c_uchar, length: ::c_int) -> c_int` (or whatever types you have in `randomNumGen.h`).